application-analytics-ui/pom.xml

```diff
@@ -0,0 +1,88 @@
+        <artifactId>application-analytics</artifactId>
+        <version>0.0.1</version>
+    </parent>
+    <artifactId>application-analytics-ui</artifactId>
+    <packaging>xar</packaging>
+    <name>Analytics  Application (Pro)</name>
+    <description>View of the app statistics. The app can be purchased individually or part of the XWiki Pro package. Try it free.</description>
+    <properties>
+        <!-- Name to display by the Extension Manager -->
+        <xwiki.extension.name>Analytics Application (Pro)</xwiki.extension.name>
+        <xwiki.extension.category>application</xwiki.extension.category>
+        <!-- The list of documents that have an implicit unlimited free license. The users can view these documents without
+         buying a license or getting a trial license, but they cannot edit or delete them. -->
+        <xwiki.extension.licensing.publicDocuments>
+        </xwiki.extension.licensing.publicDocuments>
+        <!-- The list of documents that are not covered by the extension license. The user that installs the extension has
+          full access to these documents (view, edit, delete) without having a license, so use this property wisely. -->
+        <xwiki.extension.licensing.excludedDocuments>
+        </xwiki.extension.licensing.excludedDocuments>
+        <!-- Don't run CLIRR here since there's no Java code. -->
+        <xwiki.clirr.skip>true</xwiki.clirr.skip>
+    </properties>
+    <dependencies>
+        <dependency>
+            <groupId>com.xwiki.analytics</groupId>
+            <artifactId>application-analytics-api</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.xwiki.licensing</groupId>
+            <artifactId>application-licensing-licensor-api</artifactId>
+            <version>${licensing.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.xwiki.platform</groupId>
+            <artifactId>xwiki-platform-livetable-ui</artifactId>
+            <version>${platform.version}</version>
+            <type>xar</type>
+            <scope>runtime</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.xwiki.platform</groupId>
+            <artifactId>xwiki-platform-sheet-ui</artifactId>
+            <version>${platform.version}</version>
+            <type>xar</type>
+            <scope>runtime</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.xwiki.platform</groupId>
+            <artifactId>xwiki-platform-uiextension-ui</artifactId>
+            <version>${platform.version}</version>
+            <type>xar</type>
+            <scope>runtime</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.xwiki.platform</groupId>
+            <artifactId>xwiki-platform-display-macro</artifactId>
+            <version>${platform.version}</version>
+            <scope>runtime</scope>
+        </dependency>
```

Comment:
You don't have pages that require these at the moment. Better to add a dependency when it is needed